types/recurly__recurly-js/lib/address.d.ts

```diff
@@ -1,12 +1,51 @@
 export type Address = {
+  /**
+   * Cardholder first name
+   */
   first_name: string;
+
+  /**
+   * Cardholder last name
+   */
   last_name: string;
+
+  /**
+   * First line of a street address
+   */
   address1?: string;
+
+  /**
+   * Second line of a street address
+   */
   address2?: string;
+
+  /**
+   * Town or locality
+   */
   city?: string;
+
+  /**
+   * Province or region
+   */
   state?: string;
+
+  /**
+   * Postal code
+   */
   postal_code?: string;
+
+  /**
+   * {@link http://en.wikipedia.org/wiki/ISO_3166-1_alpha-2| [ISO 3166-1 alpha-2]} country code
+   */
   country?: string;
+
+  /**
+   * 	Phone number
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/recurly__recurly-js/lib/address.d.ts;
  Line: 43;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.